src/sonic-frr/patch/series

```diff
@@ -5,3 +5,6 @@
 0005-nexthops-compare-vrf-only-if-ip-type.patch
 0007-frr-remove-frr-log-outchannel-to-var-log-frr.log.patch
 0008-Add-support-of-bgp-l3vni-evpn.patch
+0009-Link-local-scope-was-not-set-while-binding-socket-for-bgp-ipv6-link-local-neighbors.patch
+#0010-VRF-interface-lookup-was-still-done-in-the-default-vrf.patch
```

Comment:
Can you remove them instead of commenting these lines

Reply:
Done.